bodhi/server/templates/updates.html

```diff
@@ -27,6 +27,23 @@ <h3>Updates <small>
       ${tables.updates(updates, display_user, display_request)}
   % if chrome:
       ${self.pager.render(page, pages)}
+  % if active_releases:
+    <p>
+      <a href="${request.route_url('updates') + '?' \
+        + request.query_string.split('active_releases', 1)[0] \
+        + (request.query_string.split('active_releases', 1)[1].partition('&')[2] or '')}"
+         class="btn btn-sm btn-secondary">
+        Include old releases
+      </a>
+    </p>
+  % else:
+    <p>
+      <a href="${request.route_url('updates') + '?active_release=true&' + request.query_string}"
```

Comment:
Similar to what I noted below, I think this should be ```active_releases``` instead of ```active_release```. Otherwise it piles up in the URL as you click the button repeatedly.